release/models/gnsi/openconfig-gnsi.yang

```diff
@@ -1,40 +1,62 @@
 module openconfig-gnsi {
   yang-version "1";
-
+  namespace "http://openconfig.net/yang/gnsi";
   prefix "oc-gnsi";
 
-  namespace "http://openconfig.net/yang/gnsi";
+  import openconfig-extensions {
+    prefix oc-ext;
+  }
 
-  import openconfig-extensions { prefix oc-ext; }
-  import openconfig-system-grpc { prefix oc-grpc; }
+  import openconfig-system {
+    prefix oc-sys;
+  }
 
-  organization
-    "OpenConfig Working Group";
+  import openconfig-system-grpc {
+    prefix oc-sys-grpc;
+  }
+
+  import openconfig-grpc-services-common {
+    prefix oc-grpc-common;
+  }
 
+  import openconfig-gnsi-types {
+    prefix oc-gnsit;
+  }
+
+  organization
+    "OpenConfig Working group";
   contact
-    "OpenConfig working group
-    netopenconfig@googlegroups.com";
+    "www.openconfig.net";
 
   description
-    "This module defines a set of extensions that provide gNSI (the gRPC
-    Network Security Interface) specific extensions to the OpenConfig data models.
-    Specifically, the parameters for the configuration of the service, and
-    configuration and state are added.
-
-    The gNSI protobufs and documentation are published at
-    https://github.com/openconfig/gnsi.";
+    "";
 
   oc-ext:openconfig-version "0.1.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
 
-  revision 2024-02-13 {
+  revision 2024-04-23 {
     description
       "Initial revision.";
     reference "0.1.0";
   }
 
-  identity GNSI {
-    base "oc-grpc:GRPC_SERVICE";
+  grouping gnsi-base {
     description
-      "gNSI: gRPC Network Security Interface";
+      "";
+
+    container gnsi {
+      description
+        "";
+
+      uses oc-grpc-common:common-services;
+    }
+  }
+
+  augment "/oc-sys:system/oc-sys-grpc:grpc-servers" +
+          "/oc-sys-grpc:grpc-server/oc-sys-grpc:services" {
+    when "../oc-sys-grpc:config/oc-sys-grpc:services = 'oc-gnsit:GNMI'";
```

Comment:
if keeping the augment and when clause, this one should be changed to be gnsi specific

Reply:
Yep good catch - rushed out replication ...  will readjust in next patch